### DISCUSSION TITLE
I'm looking for help with some problems.

### DISCUSSION THREAD

**Discussion by sandro699** [Godiesc/firefox-gx#89] [General]
Hello to those who will read, I have installed the beautiful theme on fire fox, I have already changed the color of the theme, but I can't change the background of the main page with another image, and then how do you position the fire fox logo at the top?
![logo fire fox 1](https://github.com/Godiesc/firefox-gx/assets/148681559/f691ff0d-d3cd-4ad4-bdba-2d021dc8da73)
![sfondo fire fox](https://github.com/Godiesc/firefox-gx/assets/148681559/2cf780ff-4aff-4289-9573-8375b3b0928a)
Thanks to anyone who can give me advice - help

**Comment by Godiesc** [comment]
hi,
you can have a quick read of the main gut-hub page and found what you want.
![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/8c205f4a-cdc6-4e45-b440-8f6b23007843)
 in the same file you can change the logo and wordmark position in line 48

**Comment by sandro699** [comment]
Many thanks for your help but I display the various settings to change the appearance of the main board as shown in the photo.. there are no numbers in the lines.. can you see the point to be changed? thanks.
![Immagine 2023-10-22 184012](https://github.com/Godiesc/firefox-gx/assets/148681559/61e665a3-5cfd-43e3-8fd8-cdfc0bcfa236)
![Immagine 2023-10-22 184040](https://github.com/Godiesc/firefox-gx/assets/148681559/77944bf5-b07a-42e8-b112-8152a0f4bc1f)

**Comment by Godiesc** [comment]
You need to install a good text editor like sublime text, that helps you a lot.

**Comment by sandro699** [comment]
Hi, as you recommended, I downloaded sublime text, I'll send you the screenshot, but what can I do to put the central fire fox logo? the line I need to modify? thanks for your help.
![Immagine 2023-11-05 102406](https://github.com/Godiesc/firefox-gx/assets/148681559/6fea244b-a9d8-48c4-8b17-a780c6dff8ba)

**Comment by Godiesc** [comment]
line 47, change flex for block.
